install/install_csm_services.md

```diff
@@ -100,6 +100,14 @@ This procedure will install CSM applications and services into the CSM Kubernete
 Wait **at least 15 minutes** to let the various Kubernetes resources initialize and start before proceeding with the rest of the install.
 Because there are a number of dependencies between them, some services are not expected to work immediately after the install script completes.
 
+1. After having waited until services are healthy, take a manual backup of all Etcd clusters.
```

Comment:
Is there a way to check that "services are healthy"?

Reply:
good catch -- added command to view any pending pods...